x/auction/types/msg.go

```diff
@@ -34,9 +35,15 @@ func (msg MsgPlaceBid) Type() string { return "place_bid" }
 
 // ValidateBasic does a simple validation check that doesn't require access to state.
 func (msg MsgPlaceBid) ValidateBasic() error {
+	if msg.AuctionID == 0 {
+		return errors.New("auction id cannot be zero")
+	}
```

Comment:
What was the reason for disallowing 0 as an ID?
The default auctionID is 1 but you could set it in genesis to be 0, then the first auction created would have ID 0.

Reply:
mostly because 0 is the default value for integers (i.e uninitialized) we should avoid using 0 as a valid ID for that reason.